.circleci/config.yml

```diff
@@ -139,6 +139,28 @@ jobs:
       - store_artifacts:
           path: ./reports/screenshots
 
+  publish-docker:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:9.11.2-browsers
```

Comment:
This image doesn't have docker inside of it.
This one has:        
```
- image: circleci/node:10.19.0-buster
```
But not sure if it has everything you need?